test/test_testvectors.py

```diff
@@ -16,15 +16,19 @@ def test_testvectors():
 
 
 def check_vectors(implementation):
-    helpers.make(TYPE=implementation.scheme.type,
+    helpers.make('testvectors',
+                 TYPE=implementation.scheme.type,
                  SCHEME=implementation.scheme.name,
                  IMPLEMENTATION=implementation.name,
                  working_dir=os.path.join('..', 'test'))
     out = helpers.run_subprocess(
-        ['./testvectors_{}_{}'.format(implementation.scheme.name,
-                                      implementation.name)],
+        [os.path.join('..', 'bin', 'testvectors_{}_{}{}'.format(
```

Comment:
Shouldn't need the ../bin added in the command, as that's been set as the working dir on line 30 below.